tasks/perform_orca_reconcile/perform_orca_reconcile.py

```diff
@@ -295,7 +306,23 @@ def generate_mismatch_reports_sql() -> TextClause:
                 files.etag != reconcile_s3_object.etag OR
                 files.ingest_time != reconcile_s3_object.last_update OR
                 files.size_in_bytes != reconcile_s3_object.size_in_bytes
-            )"""
+            )"""  # nosec
```

Comment:
Why is there a `# nosec` here? Is bandit flagging something weird? Doesn't seem like it is needed.

Reply:
String injection on sql. Even though that isn't a thing anymore.

Reply:
Added basic retries. Added task https://bugs.earthdata.nasa.gov/browse/ORCA-399